### PR TITLE
Enable configurable I2C pins on esp8266

### PR DESCRIPTION
**Pull request by osos**
Title says it.

### CODE REVIEW

File: BME280.cpp [hunk 1]
```diff
@@ -207,8 +211,20 @@ BME280::BME280(uint8_t tosr, uint8_t hosr, uint8_t posr, uint8_t mode, uint8_t s
...
+  return Initialize();
+}
+#endif
+
 bool BME280::begin(){
   Wire.begin();
+  return Initialize();
+}
+
+bool BME280::Initialize() {
```
Comment (osos) [inline BME280.cpp]: Function introduced to avoid duplicate code.

**Comment by finitespace** [comment]
I will accept your pull request. Thank you for doing I2C corrections, I have been meaning to get around to it for support on esp. I have a few modifications though.

Tyler

**Comment by finitespace** [comment]
Merging pull request. Minor mods to ensue.

**Comment by finitespace** [comment]
Actually looks good. I'll leave it.

**Pull request merged**